ops_reach/instruments/methods/reach.py

```diff
@@ -337,21 +337,21 @@ def generate_metadata(header_data):
                      'SCALEMIN': 1.0e-5,
                      'SCALEMAX': 1.0,
                      'SCALETYP': 'log'}
-    meta['proton_flux1'] = {meta.labels.name: 'Proton flux from bowtie',
+    meta['proton_flux1'] = {meta.labels.name: 'Proton flux from bowtie 1',
```

Comment:
@AJHalford, is bowtie 1 / bowtie 2 sufficient? Or should this be 'from bowtie for Dosimeter 1'?